Creating new Remix workspace fails

Following the "Creating new Remix workspace" in the `@nx/remix` package:

```bash
npx create-nx-workspace@latest acme \
--preset=@nx/remix \
--project=demo
```

Fails with the following error:

```bash
✔ Successfully created the workspace: acme.

 >  NX   Cannot find module 'typescript'

   Require stack:
   - /Users/chris/Projects/acme/node_modules/@phenomnomnominal/tsquery/dist/src/ast.js
   - /Users/chris/Projects/acme/node_modules/@phenomnomnominal/tsquery/dist/src/index.js
   - /Users/chris/Projects/acme/node_modules/@nx/remix/src/utils/testing-config-utils.js
   - /Users/chris/Projects/acme/node_modules/@nx/remix/src/generators/application/lib/update-vite-test-config.js
   - /Users/chris/Projects/acme/node_modules/@nx/remix/src/generators/application/lib/index.js
   - /Users/chris/Projects/acme/node_modules/@nx/remix/src/generators/application/application.impl.js
   - /Users/chris/Projects/acme/node_modules/@nx/remix/src/generators/preset/preset.impl.js
   - /Users/chris/Projects/acme/node_modules/nx/src/config/workspaces.js
   - /Users/chris/Projects/acme/node_modules/nx/src/config/configuration.js
   - /Users/chris/Projects/acme/node_modules/nx/src/utils/package-manager.js
   - /Users/chris/Projects/acme/node_modules/nx/bin/init-local.js
   - /Users/chris/Projects/acme/node_modules/nx/bin/nx.js
   Pass --verbose to see the stacktrace.


 >  NX   Failed to apply preset: @nx/remix
```

nx report output:

```bash
 >  NX   Report complete - copy this into the issue template

   Node   : 16.19.0
   OS     : darwin arm64
   npm    : 8.19.3
   Hasher : Native
   
   nx             : 16.3.2
   @nx/js         : 16.3.2
   @nx/linter     : 16.3.2
   @nx/workspace  : 16.3.2
   @nx/devkit     : 16.3.2
   @nx/react      : 16.3.2
   @nrwl/tao      : 16.3.2
   @nx/web        : 16.3.2
   ---------------------------------------
   Community plugins:
   @nx/remix : 16.3.0
```

As you can see from that output, `typescript` is not installed. I can install it manually, but I'm not sure how to complete the project setup at that point.

--- a/packages/remix/src/utils/testing-config-utils.ts
+++ b/packages/remix/src/utils/testing-config-utils.ts
@@ -1,11 +1,17 @@
 import { stripIndents, type Tree } from '@nx/devkit';
+import { ensureTypescript } from '@nx/js/src/utils/typescript/ensure-typescript';
 import { tsquery } from '@phenomnomnominal/tsquery';
+
+let tsModule: typeof import('typescript');
 
 export function updateViteTestSetup(
   tree: Tree,
   pathToViteConfig: string,
   pathToTestSetup: string
 ) {
+  if (!tsModule) {
+    tsModule = ensureTypescript();
+  }
   const fileContents = tree.read(pathToViteConfig, 'utf-8');
 
   const ast = tsquery.ast(fileContents);
@@ -52,6 +58,9 @@ export function updateJestTestSetup(
   pathToJestConfig: string,
   pathToTestSetup: string
 ) {
+  if (!tsModule) {
+    tsModule = ensureTypescript();
+  }
   const fileContents = tree.read(pathToJestConfig, 'utf-8');
 
   const ast = tsquery.ast(fileContents);
@@ -95,6 +104,9 @@ export function updateViteTestIncludes(
   pathToViteConfig: string,
   includesString: string
 ) {
+  if (!tsModule) {
+    tsModule = ensureTypescript();
+  }
   const fileContents = tree.read(pathToViteConfig, 'utf-8');
 
   const ast = tsquery.ast(fileContents);

